src/parser/jobs/ast/index.tsx

```diff
@@ -1,8 +1,7 @@
 import {t} from '@lingui/macro'
 import {Trans} from '@lingui/react'
-import {ActionLink} from 'components/ui/DbLink'
+import {DataLink} from 'components/ui/DbLink'
```

Comment:
sounds good
I was just being consistent in changing them in case there will be other links in the future (not that it matters since this part is sanitized for EW anyway, but still)